config/rbac/role.yaml

```diff
@@ -26,6 +26,18 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts
+  verbs:
+  - '*'
+- apiGroups:
+  - ""
+  resources:
+  - services
+  verbs:
+  - '*'
```

Comment:
ok, this is a role, hence it is ONLY granted in the NROP namespace, right? Stll this is very broad, we need to narrow down AND document in the commit message why do we need these permissions

Reply:
I agree, you are right. we want in only in the NROP namespace.

Reply:
@ffromani 
I've tried to create a separate role (not clusterRole) and a RoleBinding.
But adding the label 
//+kubebuilder:rbac:groups="",resources=services,verbs=*
followed by `make manifests` appends its in the clusterRole.
Not sure how we can avoide this and append this in the dedicated role.